CakePHP/ruleset.xml

```diff
@@ -96,6 +96,8 @@
 
     <!-- Slevomat sniffs-->
     <rule ref="SlevomatCodingStandard.Arrays.TrailingArrayComma"/>
+    <rule ref="SlevomatCodingStandard.Functions.RequireTrailingCommaInDeclaration"/>
```

Comment:
Do we really need to make this required?

Reply:
They improve diff quality, same as our other ones here
They only just exist since 8.0 language, so they could not get activated earlier

Reply:
I dont think we change function declaration s often enough to require this

Reply:
But whats the issue with it, though?

Reply:
Optimal Argument count always can change for callers

Reply:
We could keep called, and ignore for signature

Reply:
It would require changing every function

Reply:
I too don't like having to change every function call.

Reply:
Changing every function call seems a bit heavy handed. These coding standard rules are also used by application code, so we'd be pushing this change onto all of those applications as well causing huge diffs there too.

Reply:
apparently it's multi-line declarations only

Reply:
Does anything handle function calls?

Reply:
Even multi-line declarations is quite a few changes across our libraries and likely in application code.